src/main/java/seedu/address/model/calendar/ReadOnlyCalendar.java

```diff
@@ -37,4 +37,13 @@ public interface ReadOnlyCalendar {
      * @return the latest ending time of any event for the current week in a LocalTime object.
      */
     Optional<LocalTime> getLatestEventEndTimeInCurrentWeek();
+
+    /**
+     * Combine this calendar with another calendar, disregarding conflicts in events.
+     *
+     * @param other other calendar to be combined.
+     * @return new calendar with events from both calendars.
+     */
+    ReadOnlyCalendar combineCalendar(ReadOnlyCalendar other);
+
```

Comment:
Perhaps better to have this method in the calendar class, since we want the ReadOnlyCalendar to be only read from, rather than have actionable methods.